src/app/clusters/service-area-server/service-area-delegate.h

```diff
@@ -96,6 +97,7 @@ class Delegate
     /**
      * @brief The server instance ensures that the SelectedAreas and CurrentArea attributes are not null before
      * calling this method.
+     * @param[in] skippedArea the area ID to skip.
```

Comment:
if this is the `skippedArea id` how about naming the variable `skippedAreaId` everywhere?

Reply:
this matches the name used by the spec

Reply:
I propose we keep inline with [the spec](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/5ebb29249385f183ff0eb8b67185311e0666efc6/src/app_clusters/ServiceArea.adoc?plain=1#L658). Maybe we can put an issue to update this in the spec.